src/utils/AxiosApi.js

```diff
@@ -3,7 +3,6 @@ import axios from "axios";
 const baseURL = axios.create({
```

Comment:
Give this 'baseUrl' a different name. "BaseUrl" doesn't tell anything about what kind of API it's calling; as you might also have multiple axios instances.
Give it a descriptive name to where it's making the request